components/Cards/tests/snapshot/Dummy.swift

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
This file is missing a copyright header - please add.

Reply:
Done